### DISCUSSION TITLE
Did something change in recent Explorer Patcher - not showing per window box in start menu

### DISCUSSION THREAD

**Discussion by nnathan** [valinet/ExplorerPatcher#3918] [Q&A] [answered]
Running `Windows Version 23H2 Build 22361.4460` and `ExplorerPatcher 22621.4317.67.1`.

I recently updated from I can't remember what version of Windows and EP.

I used to have it so that each application in the start menu such as Brave Browser, when 10 windows are open, would show as 10 boxes in the start menu for each window. Now the start menu space is empty and it shows only an aggregated icon with a blue underline to show that the application is running. There's no way to switch between the individual windows in the start menu.

Has something changed recently? I can't see any specific start menu option to change this. I have Windows 10 start menu style.

**Comment by pyrates999** [comment, marked as answer]
https://techcommunity.microsoft.com/discussions/windowsinsiderprogram/how-to-let-windows-11-never-combine-taskbar-buttons/2513043

**Reply by nnathan** [reply]
Thanks this worked.
